plankton/core/control_server.py

```diff
@@ -193,16 +196,37 @@ def __init__(self, object_map=None, host='127.0.0.1', port='10000'):
         self.port = port
```

Comment:
Yes, that message is meant for the user (to let them know they've set the `-r` option incorrectly, or that something went wrong with it). Should probably become a PlanktonException?